app/components/CounterButton/CounterButton.Styled.jsx

```diff
@@ -9,13 +9,22 @@ const handleColorType = (props) => {
   return '#31425a';
 };
 
+const setCursorType = (props) => {
+  if (props.processingFormSubmission) {
+    return 'not-allowed;';
```

Comment:
I think you don't need the semicolon. I saw where you call this function and it already has the semicolon.

Reply:
You are right about this unnecessary semicolon, I've removed it as well, thank you!